src/app/gui.html

```diff
@@ -719,8 +719,8 @@ <h2 class="subtitle">These functions may be experimental, or have only a rudimen
 					<ul>
 						<li><a id="aValidate">Validate definition</a> using <span v-if="!window.intelligentBackend"><a href="http://openapi-converter.herokuapp.com/">OpenAPI-converter</a> (online version of <a href="https://github.com/mermade/swagger2openapi">swagger2openapi</a>)</span><span v-if="window.intelligentBackend">built-in validator</span></li>
 						<li><a id="aShinola">Generate documentation</a> using <a href="https://github.com/mermade/shinola">shinola</a> (online <a href="https://github.com/mermade/widdershins">widdershins</a> and <a href="https://github.com/mermade/shins">shins</a> pipeline)</li>
-						<li><a id="aShins" href="/shins">Preview documentation</a> using <a href="https://github.com/mermade/widdershins">widdershins</a> and <a href="https://github.com/mermade/shins">shins</a> </li>
-						<li><a id="aWiddershins" href="/markdown">Dump markdown</a> using <a href="https://github.com/mermade/widdershins">widdershins</a> </li>
+						<li><a id="aShins" href="/shins" v-if="window.intelligentBackend">Preview documentation</a> using <a href="https://github.com/mermade/widdershins">widdershins</a> and <a href="https://github.com/mermade/shins">shins</a> </li>
```

Comment:
Think the `v-if` needs to go on the `<li>` not the first `<a>` tag.

Reply:
Fixed.